kubemarine/resources/configurations/defaults.yaml

```diff
@@ -151,46 +151,84 @@ services:
 
   modprobe:
     rhel:
-      - br_netfilter
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_conntrack_ipv6{% else %}nf_conntrack{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}ip6table_filter{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_nat_masquerade_ipv6{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_reject_ipv6{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_defrag_ipv6{% endif %}'
-    rhel8:
-      - br_netfilter
-      - nf_conntrack
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}ip6table_filter{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_nat{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_reject_ipv6{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_defrag_ipv6{% endif %}'
-    rhel9:
-      - br_netfilter
-      - nf_conntrack
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}ip6table_filter{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_nat{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_reject_ipv6{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_defrag_ipv6{% endif %}'
-    debian:
-      - br_netfilter
-      - nf_conntrack
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}ip6table_filter{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_nat{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_reject_ipv6{% endif %}'
-      - '{% if not nodes[0]["internal_address"]|isipv4 %}nf_defrag_ipv6{% endif %}'
+      - modulename: br_netfilter
+        groups: [control-plane, worker]
+      - modulename: nf_conntrack
+        groups: [control-plane, worker]
+        install: '{{ nodes[0]["internal_address"] | isipv4 }}'
+      - modulename: nf_conntrack_ipv6
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: ip6table_filter
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: nf_nat_masquerade_ipv6
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: nf_reject_ipv6
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: nf_defrag_ipv6
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+    rhel8: &modprobe-default-modules
+      - modulename: br_netfilter
+        groups: [control-plane, worker]
+      - modulename: nf_conntrack
+        groups: [control-plane, worker]
+      - modulename: ip6table_filter
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: nf_nat
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: nf_reject_ipv6
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
+      - modulename: nf_defrag_ipv6
+        install: '{{ nodes[0]["internal_address"] | isipv6 }}'
```

Comment:
I did not distribute the following modules among node groups:
* nf_conntrack_ipv6
* ip6table_filter
* nf_nat_masquerade_ipv6
* nf_nat
* nf_reject_ipv6
* nf_defrag_ipv6

Although they are likely `[control-plane, worker]`, I have no env to test this. Please suggest if I need to leave them as-is, or assign to Kubernetes nodes only.

Reply:
ok. let's leave as is. The ipv6 future support is TBD